ethdb/table.go

```diff
@@ -114,16 +114,11 @@ func (t *Table) Open() error {
 func (t *Table) Close() error {
 	for _, segment := range t.ldbSegments {
 		if err := segment.Close(); err != nil {
-			return err
+			log.Error("Failed to close leveldb segment", "path", segment.Path(), "name", segment.Name(), "error", err)
```

Comment:
This was failing fast on first level db segment error, and not even attempting the other types of segments.